timeseries/src/autogluon/timeseries/evaluator.py

```diff
@@ -37,6 +46,11 @@ def mape_per_item(*, y_true: pd.Series, y_pred: pd.Series) -> pd.Series:
     return ((y_true - y_pred) / y_true).abs().groupby(level=ITEMID, sort=False).mean()
 
 
+def rmsse_per_item(*, y_true: pd.Series, y_pred: pd.Series, random_walk_error: pd.Series) -> pd.Series:
```

Comment:
Is there any way that we can add a test that ensures the correctness of our RMSSE implementation over time?

Reply:
I mean we could pre-compute the RMSSE values for 1 or 2 simple examples (e.g., using a simple for-loop in the test) and add a test that ensures that the output of the TimeSeriesEvaluator matches this naive implementation.

This would ensure that in the future we don't accidentally break our RMSSE calculation in the evaluator because of some minor change.

Reply:
I added tests for this.